chapter2/tests/t20/terraform/dev/remote-exec.tf

```diff
@@ -0,0 +1,18 @@
+resource "null_resource" "execute_commands" {
+  for_each = { for idx, ec2_instance in module.ec2_instances : idx => ec2_instance }
+
+  provisioner "remote-exec" {
+    connection {
+      type        = "ssh"
+      user        = "ec2-user"
+#      private_key = file("${path.cwd}/assets/private_keys/cluster.pem")
```

Comment:
make this a habit

Reply:
> 

@dmitry-mightydevops
wrong version again
Done